lkmpg.tex

```diff
@@ -944,6 +982,11 @@ \subsection{chardev.c}
 
 \samplec{examples/chardev.c}
 
+The following codes shows the rewriting of chardev.c using cdev interface. You can compare
```

Comment:
Remove "codes"

Reply:
Always keep "one sentence one line" policy. That is, you should split the sentence starting with "You can compare" into the next line.